<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QWidget" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <widget class="QPushButton" name="certainly_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>321</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确认 | Certainly</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="account_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="tips_Label_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>账户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="tips_Label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="tips_Label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>再次输入密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirm_password_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
